docs/tutorials/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Tutorials
+nav_order: 4
+has_children: true
+---
+
+# Tutorials
```

Comment:
I'm seeing two "Tutorials" sections when I view the live docs from your branch. Is this file accidentally duplicated or something?

Reply:
Thanks. Yeah there is a duplicate Tutorials parent page. I removed one of them.